client/components/domains/domain-product-price/index.jsx

```diff
@@ -46,14 +51,30 @@ class DomainProductPrice extends React.Component {
 				}
 				break;
 			case 'INCLUDED_IN_HIGHER_PLAN':
-				if ( this.props.isEligibleVariantForDomainTest ) {
-					message = translate( 'Registration fee: {{del}}%(cost)s{{/del}} {{span}}Free{{/span}}', {
-						args: { cost: this.props.price },
-						components: {
-							del: <del />,
-							span: <span className="domain-product-price__free-price" />,
-						},
-					} );
+				if ( isEligibleVariantForDomainTest ) {
+					if ( showFreeDomainExplainerForFreePlan ) {
+						message = translate(
+							'Registration fee: {{del}}%(cost)s{{/del}} {{span}}Free with paid plan{{/span}}',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( '{{del}}%(cost)s{{/del}} {{span}}Free with a paid plan{{/span}}' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/swap-domain-plan-abtest)